browser/flagr-ui/src/components/Flag.vue

```diff
@@ -246,12 +246,12 @@
                             </el-input>
                           </el-col>
                           <el-col :span="12">
-                            <el-input
-                              size="small"
-                              placeholder="{}"
-                              v-model="variant.attachmentStr">
-                              <template slot="prepend">Attachment </template>
-                            </el-input>
+                            <vue-json-editor
+                              v-model="variant.attachment"
+                              :showBtns="false"
+                              :ref="'json-editor-' + variant.id"
```

Comment:
you can just use `:mode="'code'"`, example from https://github.com/dirkliu/vue-json-editor/blob/master/example/App.vue#L4

Reply:
You are right, it much simplier, I will change.